account_financial_risk/tests/test_account_financial_risk.py

```diff
@@ -0,0 +1,297 @@
+            "account_financial_risk.group_account_financial_risk_manager"
+        )
+        main_company = cls.env.ref("base.main_company")
+        cls.cr.execute(
+            "UPDATE res_company SET currency_id = %s WHERE id = %s",
+            [cls.env.ref("base.USD").id, main_company.id],
+        )
+        cls.account_sale = cls.env["account.account"].create(
+            {
+                "name": "Sale",
+                "code": "XX700",
+                "account_type": "income_other",
+                "reconcile": True,
+            }
+        )
+        cls.account_customer = cls.env["account.account"].create(
+            {
+                "name": "Customer",
+                "code": "XX430",
+                "account_type": "asset_receivable",
+                "reconcile": True,
+            }
+        )
+        cls.other_account_customer = cls.env["account.account"].create(
+            {
+                "name": "Other Account Customer",
+                "code": "XX431",
+                "account_type": "asset_receivable",
+                "reconcile": True,
+            }
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "Partner test",
+                "customer_rank": 1,
+                "property_account_receivable_id": cls.account_customer.id,
+                "company_id": cls.account_customer.company_ids[0].id,
+            }
+        )
+        cls.invoice_address = cls.env["res.partner"].create(
+            {
+                "name": "Partner test invoice",
+                "parent_id": cls.partner.id,
+                "type": "invoice",
+            }
+        )
+        cls.journal_sale = cls.env["account.journal"].create(
+            {
+                "name": "Test journal for sale",
+                "type": "sale",
+                "code": "TSALE",
+                "company_id": cls.env.company.id,
+            }
+        )
+        cls.tax = cls.env["account.tax"].create(
+            {
+                "name": "Tax for sale 10%",
+                "type_tax_use": "sale",
+                "tax_group_id": cls.env["account.tax.group"]
+                .with_company(cls.env.company.id)
+                .create(
+                    {
+                        "name": "Tax Group TEST",
+                    }
+                )
+                .id,
+                "amount_type": "percent",
+                "amount": 10.0,
+            }
+        )
+        cls.invoice = (
+            cls.env["account.move"]
+            .with_context(default_move_type="out_invoice")
+            .create(
+                {
+                    "partner_id": cls.partner.id,
+                    "journal_id": cls.journal_sale.id,
+                    "invoice_payment_term_id": False,
+                    "invoice_line_ids": [
+                        (
+                            0,
+                            0,
+                            {
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/370de95190cf07f17281f0ed6365b6806663cac2/odoo/fields.py#L4227-L4256) class? It is more readable. And check the remaining code, please.

```suggestion
                        Command.create(
                            {
```

Reply:
Done